web/client/utils/GeoStoryUtils.js

```diff
@@ -496,6 +496,38 @@ export const parseHashUrlScrollUpdate = (url, hash = '', storyId, sectionId, col
     return null;
 };
 
+export const customEntityTransform = (entity, text) => {
+    if (entity.type === 'MENTION') {
+        return "<a href=\"".concat(entity.data.url, "\" class=\"wysiwyg-mention\" data-mention data-value=\"").concat(entity.data.value, "\">").concat(text, "</a>");
```

Comment:
Is there a specific reason in the use of concat instead of templates?

Reply:
@allyoucanmap no I don't think so. Was just trying to be consistent with original author style.